src/app.module.ts

```
@@ -21,6 +21,7 @@ import { RbacModule } from './rbac/rbac.module';
 import { AssignTenantModule } from './userTenantMapping/user-tenant-mapping.module';
 import { FormsModule } from './forms/forms.module';
 import { HttpService } from '@utils/http-service';
+import { LMSService } from './common/services/lms.service';
```

ESLint

[error] Replace `'./common/services/lms.service'` with `"./common/services/lms.service"`

(prettier/prettier)